.github/workflows/CICD.yml

```diff
@@ -1,11 +1,6 @@
 name: CICD
 
 env:
-  PROJECT_NAME: bat
-  PROJECT_VERSION: "0.17.1"
-  PROJECT_DESC: "A `cat` clone with wings"
```

Comment:
PROJECT_DESC was unused. We need different descriptions for the Debian package due to packaging policies ("Should not start with 'A …'").